### PR TITLE
训练后量化

### PR DESCRIPTION
**Pull request by YueYongDev**
Post-training quantization翻译完成

### CODE REVIEW

**Comment by haiyang-tju** [comment]
校对认领 @YueYongDev @leviding @fanyijihua

**Comment by leviding** [comment]
@haiyang-tju Great！

File: performance/post_training_quantization.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-## Post-training quantization
+## **训练后量化**
```
Comment (haiyang-tju) [inline performance/post_training_quantization.md]: 这里的 Post-training 建议不翻译，使用原文更好理解，翻译成 训练后 不易理解。
搜了下其它对于 Post-training 的翻译方法，基本都是直接使用，不翻译~

File: performance/post_training_quantization.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
-Post-training quantization is a general technique to reduce the model size while also providing up to 3x lower latency with little degradation in model accuracy. Post-training quantization quantizes weights to 8-bits of precision from floating-point. This technique is enabled as an option in [TensorFlow Lite model converter](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/lite/toco):
+训练后量化是一种可以减小模型大小，同时降低 3 倍延迟并且仍然保持模型精度的一般方法。训练后量化将权重从浮点量化为 8 位精度。此技术在 [TensorFlow Lite model converter](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/lite/toco) 中作为一个功能选项被使用。
```
Comment (haiyang-tju) [inline performance/post_training_quantization.md]:
```suggestion
Post-training 量化是一种可以减小模型大小，同时降低 3 倍延迟并且仍然能够保持模型精度的一般方法。Post-training
 量化将权重从浮点量化为 8 位精度。此技术在 [TensorFlow Lite model converter](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/lite/toco) 中作为一个功能选项被使用。
```

File: performance/post_training_quantization.md [hunk 3]
```diff
@@ -10,43 +10,42 @@ tflite_quantized_model = converter.convert()
 open("quantized_model.tflite", "wb").write(tflite_quantized_model)
 ```
 
-At inference, weights are converted from 8-bits of precision to floating-point and computed using floating point kernels. This conversion is done once and cached to reduce latency.
+上述代码中，使用浮点内核计算将权重从 8 位精度转换为浮点数。此转换只执行一次并缓存以减少延迟。
```
Comment (haiyang-tju) [inline performance/post_training_quantization.md]:
```suggestion
在推理时，权重值从 8 位精度转换为浮点数，并使用浮点内核进行计算。此转换只执行一次并进行缓存以减少延迟。
```

File: performance/post_training_quantization.md [hunk 4]
```diff
@@ -10,43 +10,42 @@ tflite_quantized_model = converter.convert()
...
 
-To further improve latency, hybrid operators dynamically quantize activations to 8-bits and perform computations with 8-bit weights and activations. This optimization provides latencies close to fully fixed-point inference. However, the outputs are still stored using floating-point, so the speedup with hybrid ops is less than a full fixed-point computation. Hybrid ops are available for the most compute-intensive operators in a network:
+为了进一步改善延迟效果，混合运算符动态地将激活量化为 8 位，并使用 8 位权重和激活函数执行计算。此种优化方式可以提供接近完全定点推断时的延迟。但是，输出仍然使用浮点存储，因此混合运算的加速效果仍然小于完全定点计算。混合操作可用于大多数计算密集型网络：
```
Comment (haiyang-tju) [inline performance/post_training_quantization.md]:
```suggestion
为了进一步改善延迟，混合运算符动态地将激活量化为 8 位，并使用 8 位权重和激活函数执行计算。此种优化方式可以提供接近完全定点推断时的延迟。但是，输出仍然使用浮点存储，因此混合运算的加速效果仍然小于完全定点计算。混合操作可用于大多数计算密集型网络：
```

File: performance/post_training_quantization.md [hunk 5]
```diff
@@ -10,43 +10,42 @@ tflite_quantized_model = converter.convert()
...
-*  [tf.nn.embedding_lookup](https://www.tensorflow.org/api_docs/python/tf/nn/embedding_lookup)
-*  [BasicRNN](https://www.tensorflow.org/api_docs/python/tf/contrib/rnn/BasicRNNCell)
-*  [tf.nn.bidirectional_dynamic_rnn for BasicRNNCell type](https://www.tensorflow.org/api_docs/python/tf/nn/bidirectional_dynamic_rnn)
-*  [tf.nn.dynamic_rnn for LSTM and BasicRNN Cell types](https://www.tensorflow.org/api_docs/python/tf/nn/dynamic_rnn)
+- [tf.contrib.layers.fully_connected](https://www.tensorflow.org/api_docs/python/tf/contrib/layers/fully_connected)
+- [tf.nn.conv2d](https://www.tensorflow.org/api_docs/python/tf/nn/conv2d)
+- [tf.nn.embedding_lookup](https://www.tensorflow.org/api_docs/python/tf/nn/embedding_lookup)
+- [BasicRNN](https://www.tensorflow.org/api_docs/python/tf/contrib/rnn/BasicRNNCell)
+- [tf.nn.bidirectional_dynamic_rnn for BasicRNNCell type](https://www.tensorflow.org/api_docs/python/tf/nn/bidirectional_dynamic_rnn)
+- [tf.nn.dynamic_rnn for LSTM and BasicRNN Cell types](https://www.tensorflow.org/api_docs/python/tf/nn/dynamic_rnn)
```
Comment (haiyang-tju) [inline performance/post_training_quantization.md:23]: 下面的内容错行了，需要确认对应~

File: performance/post_training_quantization.md [hunk 6]
```diff
@@ -10,43 +10,42 @@ tflite_quantized_model = converter.convert()
...
 
-Since weights are quantized post-training, there could be an accuracy loss, particularly for smaller networks. Pre-trained fully quantized models are provided for specific networks in the [TensorFlow Lite model repository](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/lite/g3doc/models.md#image-classification-quantized-models){:.external}. It is important to check the accuracy of the quantized model to verify that any degradation in accuracy is within acceptable limits. There is a tool to evaluate [TensorFlow Lite model accuracy](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/lite/tools/accuracy/README.md){:.external}.
+如果精确度下降幅度过大，可以考虑使用 [quantization aware training](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/quantize/README.md)。
```
Comment (haiyang-tju) [inline performance/post_training_quantization.md]:
```suggestion
如果精确度下降幅度过大，可以考虑使用[注重量化的训练](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/quantize/README.md)方式。
```

File: performance/post_training_quantization.md [hunk 7]
```diff
@@ -10,43 +10,42 @@ tflite_quantized_model = converter.convert()
...
 </figure>
 
-The advantages of this representation format are:
+这种表示方式的好处有：
 
-* It efficiently represents an arbitrary magnitude of ranges.
-* The values don't have to be symmetrical.
-* The format represents both signed and unsigned values.
-* The linear spread makes multiplications straightforward.
+- 它有效地表示任意范围的范围。
```
Comment (haiyang-tju) [inline performance/post_training_quantization.md]:
```suggestion
- 它有效地表示任意大小的范围。
```

**Review by haiyang-tju — COMMENTED** [review]
校对完成~ @YueYongDev @leviding

**Comment by leviding** [comment]
@haiyang-tju 赞 @YueYongDev 可以修改啦

**Review by leviding — APPROVED** [review]
辛苦啦~
